src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/UpdateFrameMetadata.sql

```diff
@@ -0,0 +1,39 @@
+﻿/*************************************************************
+    Stored procedures for updating an instance status.
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     UpdateFrameMetadata
+--
+-- DESCRIPTION
+--     Update frame metadata for a list of instances matching the watermark.
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The partition key.
+--     @hasFrameMetadata
+--         * flag to indicate frame metadata existance
+--     @watermarkTableType
+--         * The list of watermarks.
+--
+-- RETURN VALUE
+--     None
+--
+CREATE OR ALTER PROCEDURE dbo.UpdateFrameMetadata
+    @partitionKey INT,
+    @hasFrameMetadata BIT,
+	@watermarkTableType dbo.WatermarkTableType READONLY
```

Comment:
Can we not just pass the `Watermark` range here instead of passing a list of watermarks?

Reply:
We can't pass the range, since some of the may not have frame data